app/src/main/java/com/nutomic/syncthingandroid/fragments/DrawerFragment.java

```diff
@@ -136,11 +136,16 @@ public void onSaveInstanceState(Bundle outState) {
      * Invokes status callbacks.
      */
     private void updateGui() {
-        if (mActivity.getApi() == null || getActivity() == null || getActivity().isFinishing())
+        if (getActivity() == null || getActivity().isFinishing()) {
```

Comment:
There is a race between checking if getActivity is null and it actually being null on the secons call, causing a NPE. Probably store the getActivitiy value in a variable.

Reply:
That's better, done with commit https://github.com/Catfriend1/syncthing-android/commit/13a180dc49c27bf80a3d1565f55dd55e93d96a0c